apps/bmd/src/AppCardlessVoting.test.tsx

```diff
@@ -1,11 +1,8 @@
 import React from 'react'
 import { fireEvent, render, waitFor, within } from '@testing-library/react'
-import { asElectionDefinition } from '@votingworks/fixtures'
-import { parseElection } from '@votingworks/types'
 import * as GLOBALS from './config/globals'
 
-// import { electionSample } from '@votingworks/types'
-import electionSample from './data/electionSample.json'
+import { electionSampleDefinition } from './data'
```

Comment:
I didn't intend to make functional changes that would require anything outside `types` to change, but I made some minor changes that seem to have required updates in BMD. So here we are.